deploy/cloud/modules/ec2/output.tf

```diff
@@ -0,0 +1,10 @@
+output "aws_instance_cloudbeat_public_ip" {
+  description = "AWS EC2 instance public IP"
+  value       = aws_instance.cloudbeat.public_ip
+}
+
+output "cloudbeat_ssh_cmd" {
```

Comment:
nice, I guess that sshing and then running kubectl will let users debug the cluster as thy wish.
Still a bit of a bummer since k9s/lens/your tool of choice will me missing there.